src/templates/homepage.html

```diff
@@ -28,6 +28,11 @@ <h2 class="hero-subtitle">Simple yet flexible JavaScript charting library for th
 					<a class="button button__green" href="https://github.com/chartjs/awesome" target="_blank">Ecosystem</a>
 					<a class="button button__black" href="https://github.com/chartjs/Chart.js" target="_blank">GitHub</a>
 				</div>
+        <!-- Meetup banner, remove after Jan 26, 2023 -->
+        <div style="margin: auto; max-width: 500px; padding-bottom: 2em; line-height: 1.25em; font-weight: bold;">
+          Join Chart.js maintainers and community at the first Chart.js meetup in January 2023. <a href="https://cube.dev/events/chart-js-community-meetup" target="_blank">Check agenda and register</a>
```

Comment:
![image](https://user-images.githubusercontent.com/39033624/207719702-4c8929ff-0e56-45f7-8f6a-75ab6e7c3ee2.png)
Seems like a dead link

Reply:
nvm did not see the comment you posted, only looked at the changes from that commit

Reply:
Yeah, and it's live already: https://cube.dev/events/chart-js-community-meetup

Reply:
👍 I put Jan 26 in my calendar. 

Anything specific you think would be good in a talk? Would a Q&A be a good idea? Maybe we can take questions ahead of time and answer them

Reply:
@etimberg Good questions! Let me take this discussion to #dev on Slack: https://chartjs.slack.com/archives/C0N98KP1N/p1671059306371829